extensions/wikia/AdEngine/js/template/bfaa.js

```diff
@@ -124,14 +130,20 @@ define('ext.wikia.adEngine.template.bfaa', [
 				return log(['show', 'not supported skin'], 'info', logGroup);
 		}
 
+		log(['show', page, wrapper, params], 'info', logGroup);
+
 		wrapper.style.opacity = '0';
 		slotTweaker.makeResponsive(params.slotName, params.aspectRatio);
 		slotTweaker.onReady(params.slotName, function (iframe) {
 			handler.show(iframe, params.aspectRatio);
 			wrapper.style.opacity = '';
+
+			if (recoveryHelper.isBlocking()) {
+				slotTweaker.tweakRecoveredSlot(iframe.parentNode.parentNode.parentNode.parentNode);
```

Comment:
I couldn't find a good way to "find" this specific element in a right way. After recovery elements get random ids and recovery library don't allow to easily find any recovered equivalent (that's why we are using hack's like this https://github.com/Wikia/app/pull/11331/files#diff-eec4847c6159c65ee41d96d4212bd940R134)